src/components/MenteeDashboard/tracks/TracksEnrollStages.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useCallback, useState } from 'react';
+import codeClanApi from '../../../api/apiUtils';
+import CustomLoader from '../../common/Spinner/CustomLoader';
+import { Radio } from 'antd';
+import tempCourseLogo from '../../assets/image/dashboard/science_image.png';
+import TracksStagesCard from './TracksStagesCard';
+
+function TracksEnrollStages({ id }) {
+  const [trackData, settrackData] = useState(null);
+  const getTrackStages = useCallback(async id => {
+    try {
+      let res = await codeClanApi.get(`tracks/${id}/stages`);
+      settrackData(res.data);
+    } catch (e) {
+      console.log(e);
+    }
+  }, []);
+  useEffect(() => {
+    getTrackStages(id);
+  }, []);
```

Comment:
Codacy found an issue: [React Hook useEffect has missing dependencies: 'getTrackStages' and 'id'. Either include them or remove the dependency array.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5838854)